internal/installation/storage/storage.go

```diff
@@ -108,12 +108,24 @@ func CachePath() string {
 		cachePath = path
 	} else {
 		cachePath = configdir.New(constants.InternalConfigNamespace, "").QueryCacheFolder().Path
-		if runtime.GOOS == "windows" {
-			// Explicitly append "cache" dir as the cachedir on Windows is the same as the local appdata dir (conflicts with config)
-			cachePath = filepath.Join(cachePath, "cache")
+	}
+
+	configRoot := filepath.Dir(cachePath)
+	entries, err := os.ReadDir(configRoot)
+	if err == nil {
```

Comment:
This is to avoid larger refactoring. I would prefer to log this error but importing the logging package here leads to an import cycle. I spent some time trying to fix it but I think that is better left for a different story. If we are going to rely on this `storage` package in many places outside of installation then we should reorganize the layout